Dapper.Tests.Performance/Massive/Massive.cs

```diff
@@ -209,7 +209,7 @@ public virtual DbConnection OpenConnection()
         /// </summary>
         public virtual List<DbCommand> BuildCommands(params object[] things)
         {
-            var commands = new List<DbCommand>();
+            var commands = new List<DbCommand>(things.Length);

```

Comment:
```suggestion
            var commands = new List<DbCommand>();
```
This is modifying Massive - a source package. It shouldn't be touched.